lnwallet/wallet.go

```diff
@@ -368,6 +368,10 @@ type LightningWallet struct {
 	intentMtx      sync.RWMutex
 	fundingIntents map[[32]byte]chanfunding.Intent
 
+	// broadcasterActive is used to determine if the broadcaster was able
+	// to be asynchronously loaded.
+	broadcasterActive atomic.Bool
```

Comment:
this could be alternatively built into the `Rebroadcaster` interface, and we can check if by calling `Started()`